internal/telemetry/metrics/grpc.go

```diff
@@ -12,93 +12,98 @@ import (
 	grpcstats "google.golang.org/grpc/stats"
 )
 
+// GRPC Views
 var (
-	grpcSizeDistribution = view.Distribution(
-		1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024,
-		2048, 4096, 8192, 16384,
-	)
-	grcpLatencyDistribution = view.Distribution(
-		1, 2, 5, 7, 10, 25, 50, 75,
-		100, 250, 500, 750, 1000,
-	)
+	// GRPCClientViews contains opencensus views for GRPC Client metrics.
+	GRPCClientViews = []*view.View{
+		GRPCClientRequestCountView,
+		GRPCClientRequestDurationView,
+		GRPCClientResponseSizeView,
+		GRPCClientRequestSizeView}
+	// GRPCServerViews contains opencensus views for GRPC Server metrics.
+	GRPCServerViews = []*view.View{
+		GRPCServerRequestCountView,
+		GRPCServerRequestDurationView,
+		GRPCServerResponseSizeView,
+		GRPCServerRequestSizeView}
 
 	// GRPCServerRequestCountView is an OpenCensus view which counts GRPC Server
 	// requests by pomerium service, grpc service, grpc method, and status
 	GRPCServerRequestCountView = &view.View{
-		Name:        "grpc_server_requests_total",
+		Name:        "grpc/server/requests_total",
```

Comment:
IIRC this gets fixed up on the exporter side so this should be okay

Reply:
Yeah -- I took this pattern from https://github.com/census-instrumentation/opencensus-go/blob/master/plugin/ochttp/stats.go#L216